content/helm/deploy.md

```diff
@@ -46,3 +48,62 @@ helm init --service-account tiller
 
 This will install **tiller** into the cluster which gives it access to manage
 resources in your cluster.
+
+### Install Demo application
+
+Clone demo application [eksdemo-chart](https://github.com/alexei-led/eksdemo-chart) chart:
+
+```sh
+# clone this repository
+git clone https://github.com/alexei-led/eksdemo-chart
```

Comment:
Let's put a `cd ~/environment` command before the git clone, just to ensure we are cloning in the correct directory.